pkg/cluster/meta/topology.go

```diff
@@ -127,6 +127,7 @@ func AllComponentNames() (roles []string) {
 // TiDBSpec represents the TiDB topology specification in topology.yaml
 type TiDBSpec struct {
 	Host            string                 `yaml:"host"`
+	ListenAddress   string                 `yaml:"listen_address"`
```

Comment:
I think it's better to set `omitempty` for this

Reply:
great, done.